### DISCUSSION TITLE
PadPlus Puppet Service was deprecated

### DISCUSSION THREAD

**Discussion by lijiarui** [wechaty/wechaty#2087] [Announcements]
亲爱的开发者，您好！

为了更好的提供服务，JuziBot Puppet Service 计划于11月30日进行一次 wechaty-puppet-padplus 重大升级。

如您正在使用 wechaty-puppet-padplus服务, 则需要进行如下升级操作。（如未使用 wechaty-puppet-padplus,可安全的忽视本通知）

wechaty-puppet-padplus 升级说明如下:
1. 卸载 wechaty-puppet-padplus, 安装 wechaty-puppet-hostie
2. 代码中启动wechaty的 wechaty-puppet-padplus 更换成 wechaty-puppet-hostie

详细说明：
wechaty-puppet-padplus 将会在11月30日下线，已付费的用户可以升级到下面的两个puppet:
1. wechaty-puppet-donut: 基于 windows 的个人微信，除了 padplus 的基础功能外，还支持接受企业微信消息，详细信息见 https://github.com/juzibot/donut-tester
2. wechaty-puppet-wxwork: 基于 windows 客户端的企业微信，详细信息见 https://github.com/juzibot/wxwork-tester

如在11月30日前没有升级代码库，wechaty-puppet-padplus 服务将会在12月1日00:00停止服务，导致您托管的微信不可用，所以强烈建议提前进行代码升级，以避免机器人意外终止服务。

如果您有任何问题，请及时与我们的客服联系，微信：`juzibot`

详情请见： https://github.com/wechaty/puppet-service-providers/issues/11

**Comment by wordgao** [comment]
支持一下！占个坑。

**Comment by huan** [comment]
Merge to https://github.com/wechaty/puppet-services/discussions/11
